sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIOTest.java

```diff
@@ -166,21 +165,12 @@ public void testReadBuildsCorrectly() {
             .withBigtableOptionsConfigurator(PORT_CONFIGURATOR);
     assertEquals("options_project", read.getBigtableOptions().getProjectId());
     assertEquals("options_instance", read.getBigtableOptions().getInstanceId());
-    assertEquals("instance", read.getBigtableConfig().getInstanceId());
-    assertEquals("project", read.getBigtableConfig().getProjectId());
+    assertEquals("instance", read.getBigtableConfig().getInstanceId().get());
+    assertEquals("project", read.getBigtableConfig().getProjectId().get());
     assertEquals("table", read.getTableId());
     assertEquals(PORT_CONFIGURATOR, read.getBigtableConfig().getBigtableOptionsConfigurator());
   }
 
-  @Test
-  public void testReadValidationFailsMissingTable() {
-    BigtableIO.Read read = BigtableIO.read().withBigtableOptions(BIGTABLE_OPTIONS);
```

Comment:
This test should still work if the user supplies `""` as the table.

Reply:
After changing validate, yes.